tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "moduleResolution": "node",
     "skipLibCheck": true,
     "lib": [
-      "es2021"
+      "es2021",
+      "ES2022.Error"
```

Comment:
`Nodejs 16` supports error's cause property, but it's included in ES2022.  We can safely pull `ES2022.Error` into our supported libs.